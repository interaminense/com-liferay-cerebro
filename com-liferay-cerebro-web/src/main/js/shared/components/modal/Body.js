import Component, {Config} from 'metal-jsx';
import getCN from 'classnames';

class Body extends Component {
	render() {
		const classes = getCN('modal-body', {
			'inline-scroller': this.props.inlineScroller
		});

		return <div class={classes}>{this.props.children}</div>;
	}
}

Body.PROPS = {
	inlineScroller: Config.bool()
};

export default Body;