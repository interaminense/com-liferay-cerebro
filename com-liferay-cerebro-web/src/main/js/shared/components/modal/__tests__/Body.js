import Body from '../Body';

describe('Modal Body', () => {
	let component;

	afterEach(() => {
		if (component) {
			component.dispose();
		}
	});

	it('should render', () => {
		component = new Body();

		expect(component).toMatchSnapshot();
	});
});