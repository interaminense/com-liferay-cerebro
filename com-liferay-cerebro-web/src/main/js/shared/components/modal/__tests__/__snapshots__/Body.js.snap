// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Body should render 1`] = `<div class="modal-body"></div>`;
